### PR TITLE
Bump minimum Java chart version

### PR DESCRIPTION
**Pull request by reespozzi**
New chart release increases the startup period for pods to stop them failing to come up in prod when a HR has been deleted / rolled back.

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by timja — CHANGES_REQUESTED** [review]
Please update the date to add another 2 weeks on given the short notice bump.
https://github.com/hmcts/cnp-jenkins-library/blob/master/vars/warnAboutDeprecatedChartConfig.groovy#L26

Also prob best to hold off and bump the node.js version too, see https://hmcts-reform.slack.com/archives/G01E6GKRKAM/p1645449797140479

**Comment by timja** [comment]
@reespozzi  reminder

**Review by reespozzi — COMMENTED** [review]
Will bump date to two weeks from today in next commit

File: resources/uk/gov/hmcts/helm/check-deprecated-charts.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 set -x
 
 CHART_DIRECTORY=${1}-${2}
-MIN_JAVA_VERSION="3.7.0"
-MIN_NODEJS_VERSION="2.4.0"
+MIN_JAVA_VERSION="3.7.1"
```
Comment (timja) [inline resources/uk/gov/hmcts/helm/check-deprecated-charts.sh:5]:
```suggestion
MIN_JAVA_VERSION="3.7.2"
```

**Review by timja — APPROVED** [review]

**Pull request merged**